### ISSUE TITLE
Error while trying to launch aerospace

### ISSUE THREAD

**Issue by RFillion** [nikitabobko/AeroSpace#370]
Here's the log : 

```
##### AeroSpace Runtime Error #####

Please report to:
    https://github.com/nikitabobko/AeroSpace/issues/new
    Please describe what you did to trigger this error

Message: Can't listen to socket /tmp/bobko.aerospace.sock The operation couldn’t be completed. (Socket.Socket.Error error 1.)
Version: 0.14.0-Beta
Git hash: 96ae7a761f5f969b21e00345629558e005d125c5
Coordinate: /Users/bobko/a/AeroSpace/Sources/AppBundle/server.swift:9:60 startServer()
recursionDetectorDuringFailure: false
cli: false

Stacktrace:
0   AeroSpace                           0x00000001024df4a8 $s6Common6errorT_4file4line6column8functionxSS_SSS2iSStlFs5NeverO_Tg5 + 740
1   AeroSpace                           0x00000001024f826c $ss6ResultO6CommonE10getOrThrow_4file4line6column8functionxSS_SSS2iSStF + 360
2   AeroSpace                           0x0000000102382064 $s9AppBundle11startServeryyF + 736
3   AeroSpace                           0x0000000102373974 $s9AppBundle04initaB0yyF + 500
4   AeroSpace                           0x000000010231f4a8 $s9AeroSpace0aB3AppV7SwiftUI0C0AadEPxycfCTW + 80
5   SwiftUI                             0x00000001b608aea8 OUTLINED_FUNCTION_0 + 124
6   AeroSpace                           0x000000010231f558 main + 36
7   dyld                                0x00000001897220e0 start + 2360
```

**Comment by nikitabobko** [comment]
Try restarting AeroSpace/macOS 🤷 #141

I've never observed this error

**Comment by RFillion** [comment]
Yes I have two users on my mac for my job and for personal use. I will restart aerospace.

**Comment by nikitabobko** [comment]
Multi-user problem should be fixed in 0.14.1. Can somebody please verify?

**Comment by arthurbellis** [comment]
I just ran into this same issue, and was thrilled to find the fix already landed. Thanks!
